src/reference/02-DetailTopics/02-Configuration/09-Macro-Projects.md

```diff
@@ -40,7 +40,7 @@ lazy val core = (project in file("core"))
     // other settings here
   )
 
-lazy val macroSub = (project in file("macro"))
+lazy val macroSub = Project("macro", file("macro"))
```

Comment:
I'd rather not use old ctor here. Could we update the example to use `macroSub` instead?